amy/settings.py

```diff
@@ -19,17 +19,16 @@
 # See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
 
 
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = json.loads(os.environ.get('DEBUG', 'true'))
-# For deployment in production: 
-# DEBUG=false SECRET_KEY="..." ./manage.py runserver ...
+# SECURITY WARNING: don't run with DEBUG turned on in production!
+DEBUG = json.loads(os.environ.get('AMY_DEBUG', 'true'))
```

Comment:
On Sun, Mar 01, 2015 at 05:47:54AM -0800, Greg Wilson wrote:
> 2015-03-01 8:28 AM, Piotr Banaszkiewicz:
> > > +DEBUG = json.loads(os.environ.get('AMY_DEBUG', 'true'))
> >
> > It's there to handle 'true' / 'false' strings, ie. convert them to
> > True/False objects in Python.
>
> That seems like a really big hammer for a really small nail - it
> would be a lot simpler, and more conformant with Unix convention, to
> simply require AMY_DEBUG=1.

It looks like the json-or-not issue has been spun off into #207.